content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -168,16 +177,34 @@ topologyKey: kubernetes.io/hostname
 whenUnsatisfiable: ScheduleAnyway
 ```
 
+{{% notice note %}}
+topologyKey refers to the key of KubeVirt infra node labels.
+{{% /notice %}}
+
 this allows us to spread Virtual Machine equally across a cluster.
 
+**Advanced settings: customize Topology Spread Constraints and Node Affinity**
```

Comment:
Same here, I think it makes sense to make it a headline. I could see

> Customize Topology Spread Constraints and Node Affinity

As the full headline here.

Reply:
done